scripts/upload-helm-charts.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -eo pipefail
+IFS=$'\n\t'
+
+bucket_name="airy-core-helm-charts"
+version=$(cat ./VERSION)
+
+case ${GITHUB_BRANCH} in
+refs/heads/develop)
+    s3_basepath=s3://${bucket_name}/stable
```

Comment:
develop is saved to `stable`?

Reply:
Correct. It will not cause a problem for the moment, as VERSION is always used when fetching the helm chart version. I opened an issue however to fix this in the future.
https://github.com/airyhq/airy/issues/2256